etest/docker/container.py

```diff
@@ -18,37 +18,39 @@ def commit(*args, **kwargs):
 
 
 def create(*args, **kwargs):
-    """Create Docker container."""
-    return common.CLIENT.create_container(*args, **kwargs)
+    """Create a Docker container."""
+    container = common.CLIENT.containers.create(*args, **kwargs)
+
+    CONTAINERS.append(container)
```

Comment:
CONTAINERS is a list ofdocker.models.containers.Container. It was already there when I started work on the migration. Elements are added and taken out by the create and remove methods, but I do not remember seeing it somewhere else in the code. It might be there tho.